Tasks/AzureAppServiceManageV0/azureappservicemanage.ts

```diff
@@ -16,6 +16,7 @@ import { enableContinuousMonitoring } from './operations/ContinuousMonitoringUti
 async function run() {
     try {
         tl.setResourcePath(path.join( __dirname, 'task.json'));
+		tl.setResourcePath(path.join( __dirname, 'module.json'));
```

Comment:
`node_modules\azure-arm-rest-v2\module.json`

Reply:
it is done